ranch/roadway.py

```diff
@@ -1101,7 +1114,7 @@ def build_centroid_connectors(
                     "Missing maz node file, will use input maz polygon centroid"
                 )
 
-                maz_node_gdf = maz_polygon_gdf["geometry"].representative_centroids()
+                maz_node_gdf = maz_polygon_gdf["geometry"].centroid
```

Comment:
Let's use [representative_point()](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.representative_point.html#geopandas.GeoSeries.representative_point) to force zone centroid to be within the polygon boundary.

Reply:
Ill have to check because my version of geopandas did not have a representative point, I will have to fiddle with environment